src/smart-app-banner/smart-app-banner.styles.ts

```diff
@@ -84,24 +85,29 @@ export const LifeSgAppIcon = styled.img`
 `;
 
 export const ButtonContainer = styled.div`
-    display: flex;
+    max-width: 30%;
     align-items: center;
 `;
 
 export const StyledButton = styled(Button.Small)`
     padding: 0.5rem;
+    height: auto;
     & > span {
-        ${TextStyleHelper.getTextStyle("XSmall", "semibold")}
+        overflow-wrap: anywhere;
+        ${TextStyleHelper.getTextStyle("XSmall", "semibold")};
     }
 `;
 
 export const Title = styled(Text.H6)`
+    overflow-wrap: anywhere;
```

Comment:
is UX ok with this style of line breaks for the title and description? are hyphens needed?

Reply:
Checked with UX, okay with `overflow-wrap: anywhere` when there is a long text without `soft hyphen` being set.

Else we can add `soft hyphen` manually in config cat which will look something like (note on the word `neighbourhood`)
```
{"title": "LifeSG mobile app","message": "Report neighbour\u00ADhood issues in 3 simple steps","buttonLabel":"Download"}
```